### DISCUSSION TITLE
Documentation Discrepancies

### DISCUSSION THREAD

**Discussion by giraffeingreen** [JerBouma/FinanceToolkit#93] [General]
I've noticed a few cases in the documentation where there appears to be a discrepancy between the function and the provided information.

For example:
- [get_delisted_stocks](https://www.jeroenbouma.com/projects/financetoolkit/docs/discovery#get_delisted_stocks) - the entire documentation seems to be from another function
- [get_weighted_average_cost_of_capital](https://www.jeroenbouma.com/projects/financetoolkit/docs/models#get_weighted_average_cost_of_capital) - the return type seems from another place

I opened a discussion with the hope that others might identify additional discrepancies and collaborate on addressing them.

I sincerely value the effort and dedication you put into your work.

**Comment by JerBouma** [comment]
Thanks, I'll correct these! And definitely if anyone finds more feel free to report it!
